include/image.mk

```diff
@@ -554,7 +554,28 @@ define Device/Build/image
 
   $(BIN_DIR)/$(call IMAGE_NAME,$(1),$(2)): $(KDIR)/tmp/$(call IMAGE_NAME,$(1),$(2))
 	cp $$^ $$@
-
+	$(if $(CONFIG_JSON_ADD_IMAGE_INFO), \
+		DEVICE_ID="$(DEVICE_NAME)" \
+		TOPDIR="$(TOPDIR)" \
+		BIN_DIR="$(BIN_DIR)" \
+		IMAGE_NAME="$(IMAGE_NAME)" \
+		IMAGE_TYPE=$(word 1,$(subst ., ,$(2))) \
+		IMAGE_SIZE="$(IMAGE_SIZE)" \
```

Comment:
I would remove IMAGE_SIZE from this patchset entirely.
We are still discussing what this variable should actually be set to and used for. Maybe we remove it in the end.
If we decide to keep or change it, we could still add it to the code again relatively simply afterwards.

Reply:
Removed